### PR TITLE
Feature/installation instructions

### PR DESCRIPTION
**Pull request by adolgert**
Steps to help our newest users:

- Created shell scripts to run Dismod-AT on the cluster or from Docker
- Documented dmcsv2db script, the one that mimics Brad's script
- Documented installation of Cascade
- Made a Pip requirements.txt file that includes the IHME db-queries requirement.

The last one is because there isn't a good way to install our software with the IHME install_extras.
The review I want is twofold.

1. Is making the shell scripts to run Dismod-AT a good idea?
2. Does Cascade installation and running of Dismod-AT with those scripts work for you?

### CODE REVIEW

**Comment by alec-deason** [comment]
We now have four separate versions of our dependency list? This can't possibly be necessary.

File: docs/install.rst [hunk 1]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
 -----------------------
 Cascade interacts with Dismod-AT underneath. Cascade runs Dismod-AT within
 the IHME infrastructure. Clone it from
```
Comment (nrhawkins) [inline docs/install.rst]: It's maybe a little confusing to read "Clone it" here, as well as on line 12.

File: docs/install.rst [hunk 2]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
-Python environment or virtual environment.
+`Cascade on Github <https://github.com/ihmeuw/cascade>`_.
+We recommend you create a virtual environment into which to install
+the code. Then::
 
+    git clone https://github.com/ihmeuw/cascade.git
+    # Or use the one below if you have a Github account.
+    # git clone git@github.com:ihmeuw/cascade.git
+    cd cascade
+    pip install .[ihme_requirements,testing]
```
Comment (nrhawkins) [inline docs/install.rst]: Should this line (16) be typed exactly as written?

File: docs/install.rst [hunk 3]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
+    python setup.py develop
+    cd tests && pytest
```
Comment (nrhawkins) [inline docs/install.rst]: These worked for me.

File: docs/install.rst [hunk 4]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
 
 Installation of Dismod-AT
 -------------------------
+Dismod-AT is already installed on the cluster. If you've installed the
+Cascade as described above, it will have created two commands,
+``dmdismod`` and ``dmdismodpy``, which are the application and its
+Python helper. You can run, for instance::
+
+    dmdismod data.db init
+    dmdismod data.db fit
```
Comment (nrhawkins) [inline docs/install.rst]: (env) [nhawkins@cn497 cascade]$ dmdismod data.db fit
dismod_at-20180807 release build
fit command expected 1 or 2 arguments to follow fit

File: docs/install.rst [hunk 5]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
+    dmdismod data.db predict
```
Comment (nrhawkins) [inline docs/install.rst]: (env) [nhawkins@cn497 cascade]$ dmdismod data.db predict
dismod_at-20180807 release build
predict command expected 1 arguments to follow predict

File: docs/install.rst [hunk 6]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
+    dmdismodby data.db db2csv
```
Comment (nrhawkins) [inline docs/install.rst]: dmdismodby -> dmdismodpy

File: docs/install.rst [hunk 7]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
+
+Without the helper, the same commands would be::
```
Comment (nrhawkins) [inline docs/install.rst]: I like the helper commands.

File: docs/install.rst [hunk 8]
```diff
@@ -5,31 +5,77 @@ Installation of Cascade
...
 
 It's a long command, but it will run Dismod-AT.
+
+If you use the dmdismod commands and have ``data.db`` in the
+current directory, you would run::
+
+    dmdismod /app/data.db init
+    dmdismod /app/data.db fit
+    dmdismod /app/data.db predict
+    dmdismodby /app/data.db db2csv
```
Comment (nrhawkins) [inline docs/install.rst]: dmdismodby -> dmdismodpy

**Pull request merged**